index.js

```diff
@@ -44,6 +44,10 @@ const whitelistedAccounts = (process.env.WHITELISTED_ACCOUNTS || '')
   .toLowerCase()
   .split(',');
 
+const checkAfter24Hrs = require(
+  './lib/pingCodeOwnerAfter24hrs'
+);

```

Comment:
Move this more up with the other imports.

Reply:
Ok Sir.